sync/src/main/java/tech/pegasys/artemis/sync/SyncManager.java

```diff
@@ -51,6 +51,8 @@
   private final AsyncRunner asyncRunner;
   private final Set<NodeId> peersWithSyncErrors = new HashSet<>();
 
+  private UnsignedLong highestSlot = UnsignedLong.ZERO;
```

Comment:
I wouldn't have expected us to store this.  Just `findBestSyncPeer` in `getSyncStatus` so we ensure we have an up to date value regardless of which peer we are currently syncing to.

Reply:
fixed